### PR TITLE
[DAQmx] Added optional paramaters for enums and attributes metadata

### PR DESCRIPTION
**Pull request by MounikaBattu17**
### What does this Pull Request accomplish?

- Added optional parameters '`python_description`' and '`python_enum`' enums schema and attributes schema in `metadata_validation.py` to facilitate the newly updated Scrapigen metadata [PR](https://github.com/ni/grpc-device-scrapigen/pull/146)

### Why should this Pull Request be merged?

- To maintain the gRPC device compatibility with the updated Scrapigen metadata

### What testing has been done?

- Regenerated proto files in `generated/nidaqmx` folder
- Verified that no updates in `generated` folder

### CODE REVIEW

File: source/codegen/metadata/nidaqmx/enums.py [hunk 1]
```diff
@@ -685,21 +686,24 @@
...
+                    'description': ' Sensor is a full bridge. If you set DAQmx_AI_Excit_UseForScaling to True,  NI-DAQmx divides the measurement by the excitation value. Many sensors scale  data to native units using scaling of volts per excitation.',
+                    'python_description': 'Sensor is a full bridge. If you set **ai_excit_use_for_scaling** to True, NI-DAQmx divides the measurement by the excitation value. Many sensors scale data to native units using scaling of volts per excitation.'
                 },
                 'name': 'FULL_BRIDGE',
                 'value': 10182
             },
             {
                 'documentation': {
-                    'description': ' Sensor is a half bridge. If you set DAQmx_AI_Excit_UseForScaling to TRUE,  NI-DAQmx divides the measurement by the excitation value. Many sensors scale  data to native units using scaling of volts per excitation.'
+                    'description': ' Sensor is a half bridge. If you set DAQmx_AI_Excit_UseForScaling to True,  NI-DAQmx divides the measurement by the excitation value. Many sensors scale  data to native units using scaling of volts per excitation.',
```
Comment (bkeryan) [inline source/codegen/metadata/nidaqmx/enums.py:697]: This includes unwanted changes to the C/CVI descriptions (e.g. `TRUE` -> `True`). When that is resolved in the scrapigen PR, please update this PR.

File: source/codegen/metadata/nidaqmx/attributes.py
Comment (bkeryan) [inline source/codegen/metadata/nidaqmx/attributes.py:1]: Why does GitHub think almost every line changed? Only one line changed. Is this a line ending issue?

File: source/codegen/metadata/nidaqmx/enums.py (hunk 1 above)
Comment (MounikaBattu17) [reply source/codegen/metadata/nidaqmx/enums.py]: Updated the expected enums.py from Scrapigen https://github.com/ni/nidaqmx-python/pull/215

**Review by bkeryan — APPROVED** [review]

**Comment by MounikaBattu17** [comment]
@zhindes @astarche Brad has approved the PR. can you please provide your comments?

**Comment by zhindes** [comment]
> @zhindes @astarche Brad has approved the PR. can you please provide your comments?

I don't typically look a the grpc-device PRs; I trust these folks! Is there something specific you want me to look at? If not, I'll let them handle it :)

**Review by reckenro — APPROVED** [review]
Looks good to me! And Alex (astarche) is out today and said it's fine to bypass him so feel free to submit after the build completes.

**Pull request merged**